config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -538,21 +538,21 @@ periodics:
         securityContext:
           privileged: true
         env:
-        - name: TEST_CCM
+        - name: TEST_CCM # CAPZ config
           value: "true"
-        - name: AZURE_LOADBALANCER_SKU
+        - name: AZURE_LOADBALANCER_SKU # cloud-provider-azure config
           value: "standard"
-        - name: CONTROL_PLANE_MACHINE_COUNT
+        - name: CONTROL_PLANE_MACHINE_COUNT # CAPZ config
           value: "1"
-        - name: TEST_WINDOWS
+        - name: TEST_WINDOWS # CAPZ config
           value: "true"
-        - name: WINDOWS_FLAVOR
+        - name: WINDOWS_FLAVOR # I am not sure that this is actually doing anything? @jackfrancis
```

Comment:
cc @marosset @jsturtevant do you know what's going on here?

Also Mark and I noticed that a recent successful run from this job is using Windows 2019, which may not be what we want:

https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/cloud-provider-azure-ccm-windows-capz/1619002501678239744

Reply:
When I added this job, I followed another Windows job and followed this env var. Maybe it should be updated.

Reply:
I think at one point provisioning clusters with Windows Server 2019 or Windows Server 2022 nodes required specifying different CAPZ templates. the `WINDOWS_FLAVOR` vars might have been from that?

Anyways this is the best way to specify Windows Server 2022 nodes

```
          - name: WINDOWS_SERVER_VERSION # CAPZ config
              value: "windows-2022"
```

@jackfrancis do you want to update this PR or should I make a follow-up PR to remove `WINDOWS_FLAVOR` from all the configs in test-infra?

Reply:
Agree w/ Mark about what the fix is. IMO we should just do this in this PR.